.gitignore

```diff
@@ -1,6 +1,7 @@
 # Files
 *.txt
 scippconfig.py
```

Comment:
Is scippconfig.py still used after it is replaced in master.yml above?

Reply:
It's gone. I'll remove it.